framework_tvb/tvb/interfaces/web/controllers/simulator_controller.py

```diff
@@ -239,6 +240,10 @@ def _update_last_loaded_fragment_url(self, current_url):
         self.last_loaded_form_url = current_url
         common.add2session(common.KEY_LAST_LOADED_FORM_URL, self.last_loaded_form_url)
 
+    @cherrypy.expose
+    def cancel_or_remove_burst(self, burst_id):
+        ProjectService().cancel_or_remove_burst(burst_id)
```

Comment:
On the client-side, I see it expects an answer from this method.
Please check how this method handled results in the old tvb version:
https://github.com/the-virtual-brain/tvb-framework/blob/0db0f63c3f5c5485063cda014aa8698e23f709df/tvb/interfaces/web/controllers/burst/burst_controller.py#L423